aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -24,3 +24,13 @@ proposals:
           git_hash: ~
       - DefaultGasWithOverride:
           feature_version: 17
+  - name: step_3_enable_remove_detailed_error_from_hash
+    metadata:
+      title: "Enable debug info removal"
+      description: "turn on feature flag to remove the vm debug info from transaction info"
+    execution_mode: MultiStep
+    update_sequence:
+      - FeatureFalg:
+          enabled:
+            - remove_detailed_error_from_hash
+  
```

Comment:
this shouldn't be committed, this needs to be separate from the "framework only" release, as far as I understand